gruntfile.js

```diff
@@ -29,13 +29,13 @@ module.exports = function (grunt) {
                 karmaTests: 'tests/unit/**/*.js'
             }
         },
-        jshint: {
-            options: {
-                jshintrc: '.jshintrc'
-            },
-            all: [
-                '<%= config.files.lint %>'
-            ]
+        eslint: {
+            all: {
+                options: {
+                    config: 'eslint.json'
+                },
+                src: '<%= config.files.lint %>'
+            }
         },
         jscs: {
```

Comment:
this is an upcoming feature and isnt fully ready yet.  @grahamgilchrist  anything else i can do?